pom.xml

```diff
@@ -24,7 +24,7 @@
     <alpn.api.version>1.1.3.v20160715</alpn.api.version>
     <java.level>8</java.level>
     <!-- TODO: remove once SpotBugs issues are fixed. 57 so far -->
```

Comment:
Still valid?

Reply:
29 issues left now :-) I'll update the number.